lib/BillTechLinkApiService.php

```diff
@@ -101,17 +101,19 @@ public static function getLinkData($linkRequest)
 		if ($linkRequest->srcDocumentId) {
 			// TODO: use customercontacts for email
 			$linkData = $DB->GetRow("select" . ($DB->GetDbType() == "postgres" ? " distinct on (c.id)" : "") .
-				" d.customerid, d.number, concat(d.fullnumber, case when d.comment is not null then concat(' ', d.comment) else '' end) as comment, 
-       									d.id, c.lastname, c.name, d.cdate, d.paytime, di.id as division_id, di.shortname as division_name, cc.contact as email from documents d
+				" d.customerid, d.number, d.fullnumber, d.comment as comment_string, d. d.div_account, d.id, c.lastname,
+				 c.name, d.cdate, d.paytime, di.id as division_id, di.shortname as division_name, cc.contact as email 
+				 						from documents d
     									left join customers c on d.customerid = c.id
        									left join customercontacts cc on cc.customerid = c.id and (cc.type & 8) > 1
 										left join divisions di on c.divisionid = di.id where d.id = ?" . ($DB->GetDbType() == "mysql" ? " group by c.id" : ""), [$linkRequest->srcDocumentId]);
 			if (!$linkData) {
 				throw new Exception("Could not fetch link data by document id: " . $linkRequest->srcDocumentId);
 			}
+			$linkData['comment'] = self::getDocumentComment($linkData['fullnumber'], $linkData['comment_string'], $linkData['name']);
```

Comment:
Wcześniej było `d.name` teraz jest `c.name`. To jest to samo?

Reply:
c to tabela customer, d to document. d.name to imię klienta, więc uznałem że c.name to to samo. Niestety w tabeli document w przypadku encji z fullnumber=null i comment=null średnio jest z czego brać nazwę faktury dlatego "Faktura Imię Nazwisko". Jest to rozwiązanie tymczasowe, żeby nie leciały błędy przy generowaniu faktur.

Reply:
Ok, trochę to dziwne, że w tabeli documents jest imię klienta, ale jak jesteś tego pewien to git.